starters/deno-oak-denodb/src/main.ts

```diff
@@ -1,24 +1,38 @@
-import { Application, config, Context, oakCors, Router } from '../deps.ts';
+import { Application, applyGraphQL, config, Context, oakCors, Router } from '../deps.ts';
+import { db } from './db/db.ts';
+import { technologyResolvers } from './db/resolvers/technology.ts';
+import { technologyTypes } from './db/schema/technology.ts';
 
 const { API_HOST, CORS_ALLOWED_ORIGINS, PORT } = config({ safe: true });
 const app = new Application();
 const port = +PORT || 3333;
 
 const router = new Router();
 router.get('/', ({ request, response }: Context) => {
-  response.body = `Hello world! from ${request.url}`;
+	response.body = `Hello world! from ${request.url}`;
+});
+
+const GraphQLService = await applyGraphQL<Router>({
+	Router,
+	typeDefs: technologyTypes,
+	resolvers: technologyResolvers,
 });
 
 app.use(
-  oakCors({
-    allowedHeaders: ['Content-Type', 'Authorization'],
-    origin: CORS_ALLOWED_ORIGINS ? CORS_ALLOWED_ORIGINS.split(',') : '',
-    optionsSuccessStatus: 200, // Some legacy browsers (IE11, various SmartTVs) choke on 204
-  }),
+	oakCors({
+		allowedHeaders: ['Content-Type', 'Authorization'],
+		origin: CORS_ALLOWED_ORIGINS ? CORS_ALLOWED_ORIGINS.split(',') : '',
+		optionsSuccessStatus: 200, // Some legacy browsers (IE11, various SmartTVs) choke on 204
+	}),
 );
+
+app.use(GraphQLService.routes(), GraphQLService.allowedMethods());
 app.use(router.routes());
 app.use(router.allowedMethods());
 
+await db.sync({ drop: true });
```

Comment:
This is a dangerous operation to be in the app startup module, especially with `drop: true`. I would suggest to place it in its own module.

Reply:
For example, you can place it in `seed.ts` and then the seed command would be `deno run seed.ts ...`. It's a manual command that needs to be executed during project setup.

Reply:
Oh, I see you already have `run_seeder.ts`. Yeah, that's good.

So you can remove this line then.